manifests/params.pp

```diff
@@ -22,5 +22,5 @@
   $user = 'qpidd'
   $group = 'qpidd'
 
-  $server_packages = ['qpid-cpp-server', 'qpid-cpp-client', 'python-qpid-qmf', 'python-qpid', 'policycoreutils-python', 'qpid-tools']
+  $server_packages = ['qpid-cpp-server', 'qpid-cpp-client', 'python-qpid-qmf', 'python-qpid', 'policycoreutils-python', 'qpid-tools', 'qpid-cpp-server-linearstore']
```

Comment:
Looks like it is in epel 7:
https://dl.fedoraproject.org/pub/epel/7/x86_64/repoview/qpid-cpp-server-store.html

On Tue, Aug 4, 2015 at 10:36 PM, Stephen Benjamin <notifications@github.com>
wrote:

> In manifests/params.pp
> <https://github.com/Katello/puppet-qpid/pull/26#discussion_r36265118>:
>
> > @@ -22,5 +22,5 @@
> >    $user = 'qpidd'
> >    $group = 'qpidd'
> >
> > -  $server_packages = ['qpid-cpp-server', 'qpid-cpp-client', 'python-qpid-qmf', 'python-qpid', 'policycoreutils-python', 'qpid-tools']
> > +  $server_packages = ['qpid-cpp-server', 'qpid-cpp-client', 'python-qpid-qmf', 'python-qpid', 'policycoreutils-python', 'qpid-tools', 'qpid-cpp-server-linearstore']
>
> Is it still in EPEL? I think it makes sense, this should probably be
> parameter and give the option of providing a store or not (same for things
> like installing the client, tools, etc).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Katello/puppet-qpid/pull/26/files#r36265118>.
>